proposals/list-patterns.md

```diff
@@ -43,7 +43,7 @@ property_pattern
   ;
 
 slice_pattern
-  : '..' negated_pattern?
+  : '..' pattern?
```

Comment:
Wasn't mentioned in LDM. But I think "all patterns" includes combinators.

Reply:
It does. I'm not familiar enough with the various levels of the pattern grammar, does `negated_pattern` not include combinators?

Reply:
No it doesn't. I've started this as an unary operator, hence the precedence.

Reply:
nit: Consider adding links to previous patterns specs:

https://github.com/dotnet/csharplang/blob/main/proposals/csharp-7.0/pattern-matching.md
https://github.com/dotnet/csharplang/blob/main/proposals/csharp-8.0/patterns.md
https://github.com/dotnet/csharplang/blob/main/proposals/csharp-9.0/patterns3.md